odometry_topic: /odometry/ground_truth
aggregation_time_topic: /aggregation_time
aggregation_time_seconds: 1
